defaults/main.yml

```diff
@@ -8,6 +8,7 @@ varnish_default_vcl_template_path: default.vcl.j2
 varnish_default_backend_host: "127.0.0.1"
 varnish_default_backend_port: "8080"
 
+varnish_listen_address: ""
 varnish_listen_port: "80"
```

Comment:
These are from the upstream repo, so for potential pull-request purposes I didn't want to make changes like that.